src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -206,6 +207,17 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
     return [];
   }
 
+  override async fetchTokenInfo(): Promise<IToken> {
+    // TODO if needed
+    // @ts-expect-error
```

Comment:
I think the base protocol adapter should allow to return undefined because we know that some protocols has no tokens (BTC).